Demos/Blazorise.Demo.AntDesign/Properties/launchSettings.json

```diff
@@ -20,10 +20,10 @@
       "commandName": "Project",
       "launchBrowser": true,
       "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "applicationUrl": "http://localhost:7715/",
```

Comment:
Why did you change all the `launchSettings.json` files?

Reply:
I compiled it like i usually do. I do not touch any other files